.opencode/skills/scripts/check-rust-quality.sh

```
@@ -0,0 +1,50 @@
+if [ "$UNWRAP_COUNT" -gt 0 ]; then
+  echo "❌ FAIL: $UNWRAP_COUNT unwrap() calls found in production code"
+  grep -rn '\.unwrap()' notescribe-core/src/ --include='*.rs' --exclude='*test*' --exclude='*tests*' | grep -v '// SAFETY:'
+  exit 1
+fi
+
+# 3. Check for unsafe without SAFETY comment
+echo "  → Checking unsafe usage..."
+UNSAFE_COUNT=$(grep -rn 'unsafe' notescribe-core/src/ \
+  --include='*.rs' \
+  | grep -v '// SAFETY:' \
+  | grep -v 'Cargo.toml' \
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)